wolfengine/README.md

```diff
@@ -0,0 +1,42 @@
+# wolfSSL Open Source Project Ports with wolfEngine
+
+The goal of the wolfEngine project is to replace the crypto used by OpenSSL
+with wolfCrypt or wolfCrypt FIPS, with the least amount of changes possible.
+
+Each of the project directories has both a patch file and a reference to a
+submodule for the project at the commit for the patch. After cloning the OSP
+repository, change directories into the project's subdirectory. Run the
+command `git submodule update && git submodule update`. Or if the OSP
```

Comment:
I think you mean `git submodule init && git submodule update`. For some reason, when I pull down this PR as a patch and apply it, the submodule commands aren't doing anything:

```
hayden@hayden-dell: /home/hayden/wolfssl_osp/wolfengine/libssh$ ls -l
total 8
---------- 1 hayden hayden  59 Jan  4 07:41 libssh
-rw-r--r-- 1 hayden hayden 624 Jan  4 07:41 libssh-patch.txt
hayden@hayden-dell: /home/hayden/wolfssl_osp/wolfengine/libssh$ git submodule init && git submodule update
hayden@hayden-dell: /home/hayden/wolfssl_osp/wolfengine/libssh$ ls -l
total 8
---------- 1 hayden hayden  59 Jan  4 07:41 libssh
-rw-r--r-- 1 hayden hayden 624 Jan  4 07:41 libssh-patch.txt
```

Reply:
@haydenroche5 `git submodule init && git submodule update` should be run from the inner-most project directory. The wording in the instructions could be changed to make that more clear.

Reply:
I don't think that's the problem:

Reply:
Oh, I see the same thing when applying the PR as a patch. Not sure why it's not working but fetching then checking out to the PR worked fine

Reply:
I updated the directions there. It should have been init then update, but you do it anywhere in the repo directory it updates them all.

It looks like you have to actually check out the project or branch. If you do it with a patch it doesn't work. Just inserting a `.gitmodules` file to a repo doesn't seem to do the submodule setup.